components/feature/toolbar/src/main/java/mozilla/components/feature/toolbar/WebExtensionToolbarFeature.kt

```diff
@@ -94,6 +94,10 @@ class WebExtensionToolbarFeature(
     internal fun renderWebExtensionActions(state: BrowserState, tab: SessionState? = null) {
         val extensions = state.extensions.values.toList()
         extensions.filter { it.enabled }.sortedBy { it.name }.forEach { extension ->
+            if (!extension.allowedInPrivateBrowsing && tab?.content?.private == true) {
```

Comment:
`override fun start()` needs the same filter to remove any stale actions..

Reply:
After allowing  UBlock in private the [toolbars is not getting updates](https://drive.google.com/file/d/1PLTYjd1f-ufrkzWnPIqwkRiFnOuOWJ62/view?usp=sharing) while the menu is.

Reply:
@Amejia481 updated!